server/utils/userUtils.test.ts

```diff
@@ -2,9 +2,28 @@ import { sectionsForUser, sections } from './userUtils'
 
 describe('userUtils', () => {
   describe('sectionsForUser', () => {
+    it('should return an empty array for a user with no roles', () => {
+      expect(sectionsForUser([])).toEqual([])
+    })
+
     it('should return correct sections for a POM', () => {
       const expected = [sections.applications, sections.newApplication]
-      expect(sectionsForUser()).toEqual(expected)
+      expect(sectionsForUser(['POM'])).toEqual(expected)
+    })
+
+    it('should return the prison dashboard for a Licence CA', () => {
+      const expected = [sections.applications, sections.newApplication]
+      expect(sectionsForUser(['LICENCE_CA'])).toEqual(expected)
+    })
+
+    it('should return correct sections for an admin', () => {
+      const expected = [sections.submittedApplications]
+      expect(sectionsForUser(['CAS2_ADMIN'])).toEqual(expected)
```

Comment:
These roles don't seem to have the `ROLE_` prefix, was this the issue you mentioned encountering?

Reply:
I think the new TS template has updated that, so setUpCurrentUser gets rid of the prefix: https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-bail-ui/blob/77c3c8756a66708ca6f0e4c1e20eeabbf331f894/server/middleware/setUpCurrentUser.ts#L26

Also the key is now called `userRoles`, rather than `roles`.